homework_2/fastq_filtrator.py

```diff
@@ -0,0 +1,76 @@
+def gc_in_bounds(seq: str, gc_bounds=(0, 100)):
+    """gc_in_bounds(seq, gc_bounds=(0, 100))
+
+    Counts gc_content for the sequence (seq) and determines if the received value is in gc_bounds.
+    """
+    gc_content = (seq.count('G') + seq.count('C')) / len(seq) * 100
+    if isinstance(gc_bounds, int) and gc_content <= gc_bounds:
+        return True
+    elif isinstance(gc_bounds, tuple) and gc_bounds[0] <= gc_content <= gc_bounds[1]:
+        return True
+    else:
+        return False
+
+
+def len_in_length_bounds(seq, length_bounds=(0, 2**32)):
+    """len_in_length_bounds(seq, length_bounds=(0, 2**32))
+    Counts length of the sequence (seq) and determines if the received value is in length_bounds.
+    """
+    if isinstance(length_bounds, int) and len(seq) <= length_bounds:
+        return True
+    elif isinstance(length_bounds, tuple) and length_bounds[0] <= len(seq) <= length_bounds[1]:
+        return True
+    else:
+        return False
+
+
+def mean_quality_in_threshold(quality_line, quality_threshold=0):
+    """mean_quality_in_threshold(quality_line, quality_threshold=0)
+        Counts mean value of the quality_line and determines if the received value is greater than or equal to
+        quality_threshold.
+        """
+    mean_quality = sum([ord(i) - 33 for i in quality_line]) / len(quality_line)
+    if mean_quality >= quality_threshold:
+        return True
+    else:
+        return False
```

Comment:
Можно сильно упростить
```suggestion
    return mean_quality >= quality_threshold
```